packages/guilded.js/lib/structures/Message.ts

```diff
@@ -85,6 +86,11 @@ export class Message extends Base<ChatMessagePayload> {
         return this.createdByBotId ?? this.createdByWebhookId ?? this.createdById;
     }
 
+    /** Get the member of this message (if in server) */
+    get member(): Member | null {
```

Comment:
Do we want to make on universal function for constructing server member cache keys?

Reply:
That would be ideal!

Reply:
Just implemented this in https://github.com/guildedjs/guilded.js/pull/75/commits/7da1b13543f246c8e600777aeb36f3646b1aba4e 😄